docs/deploy.rst

```diff
@@ -0,0 +1,228 @@
+
+.. _deploy:
+
+======
+Deploy
+======
+
+To deploy a CrateDB cluster, you first need an organization and project, and
+this page takes you through each of the steps. You will:
+
+- Create an organization
```

Comment:
please collapse this into one or more paras in a more narrative style (not list style)